src/common/make-enum.ts

```diff
@@ -191,21 +200,19 @@ type ValuesOfDeclarations<ValueDeclaration extends EnumValueDeclarationShape> =
  * properties as optional.
  */
 type NormalizedValueDeclaration<Declaration extends EnumValueDeclarationShape> =
-  SetRequired<Pick<EnumValueDeclarationObjectShape<any>, 'label'>, 'label'> &
```

Comment:
Essentially just moved this from here, which is only referenced in output of `makeEnum` to [new line 19](https://github.com/SeedCompany/cord-api-v3/pull/3225/files#diff-5d4630304550842a4c2a9d27a4019353b81c45c87a67a02eceeba07a0682cff2R19) within `MadeEnum`.

This allows others to use the `MadeEnum` type and get the implicits that were previously only available from the return type of `makeEnum`